package.json

```diff
@@ -51,7 +51,8 @@
   },
   "private": true,
   "dependencies": {
-    "espree": "5.0.1"
+    "espree": "5.0.1",
+    "kolibri-design-system": "https://github.com/learningequality/kolibri-design-system"
```

Comment:
This will ultimately target the published package but for now - testing it with the master branch works.